src/jcstress/java/reactor/pool/SimpleDequePoolStressTest.java

```diff
@@ -0,0 +1,105 @@
+/*
+ * Copyright (c) 2018-Present Pivotal Software Inc, All Rights Reserved.
```

Comment:
nit: VMware

Reply:
the license plugin should either be removed or all the license headers should be updated. will be done another time in either way